app/lib/features/chat_ng/widgets/chat_editor/chat_editor.dart

```diff
@@ -219,7 +219,7 @@ class _ChatEditorState extends ConsumerState<ChatEditor> {
     if (text.isEmpty || actualLineCount < 2) {
       newHeight = ChatEditorUtils.baseHeight;
     } else {
-      final offset = 30 * (actualLineCount - 1);
+      final offset = 15 * (actualLineCount - 1);
```

Comment:
set it up to half of the previous value (because of newline adjustments), so it doesn't expand more than it should on newline.